.clang-tidy

```
@@ -1,5 +1,5 @@
 ---
-Checks: "-*,modernize-use-bool-literals,modernize-use-nullptr,modernize-use-override,readability-braces-around-statements,readability-namespace-comment,readability-non-const-parameter,readability-qualified-auto"
+Checks: "-*,modernize-use-bool-literals,modernize-use-nullptr,modernize-use-override,readability-braces-around-statements,google-readability-namespace-comments,readability-non-const-parameter,readability-qualified-auto"
```

```suggestion
Checks: >
  -*,
  modernize-use-bool-literals,
  modernize-use-nullptr,
  modernize-use-override,
  readability-braces-around-statements,
  google-readability-namespace-comments,
  readability-non-const-parameter,
  readability-qualified-auto,
```
the line has become quite long